apps/src/studio/studio.js

```diff
@@ -1233,20 +1235,17 @@ function handleActorCollisionsWithCollidableList (
             // NOTE: if items are allowed to move outOfBounds(), this may never
             // be called because the last item may not be removed here.
 
-            if (list.length === 1) {
+            if (activeCollidables.length === 1) {
```

Comment:
Unfortunately, down here where we check whether we are collecting the last item, we look at the size of the whole list... which may include items that are currently fading out.  Therefore, the following case breaks:

1. Make a level with two stormtroopers.
2. Collect stormtrooper #1.  This logic runs, but `list.length` is 2 so nothing happens.
3. Collect stormtrooper #2 while the first one is still fading out.  This logic runs, but `list.length` is still 2, so nothing happens.
4. Both stormtroopers fade out, but the `whenGetAllItems` handlers never runs.